components/raftstore/src/store/fsm/store.rs

```diff
@@ -561,6 +561,9 @@ where
             self.cfg.check_long_uncommitted_interval.0;
         self.tick_batch[PeerTick::CheckPeersAvailability as usize].wait_duration =
             self.cfg.check_peers_availability_interval.0;
+        // TODO: make it reasonable
+        self.tick_batch[PeerTick::RequestVoterReplicatedIndex as usize].wait_duration =
```

Comment:
The index only makes sense when there are pending log gc. The tick can be register on demand.

Reply:
BTW,  should the Pending Compact cmd need to be persisted? eg: After several cmds were pending, a reboot occurred.

Reply:
Technically it should. But it can be avoided by utilizing `RequestVoterReplicatedIndex`. Leader may return its first index as well as replicated index, so witness can gc logs even after being restarted.